connexion/operations/openapi.py

```diff
@@ -273,7 +273,7 @@ def _get_body_argument(self, body, arguments, has_kwargs, sanitize):
 
         if x_body_name in arguments or has_kwargs:
```

Comment:
@kornicameister, you and I are both trying to get aiohttp to work well with connexion. I can't wait until this PR is accepted.

Possibly another thing to add in this area of change: 
Can you at all fathom why the 'or has_kwargs' is here???  I'm trying to create decorators for my handlers.  The decorators generically pass all args along as **kwargs.  However, when **kwargs is present in the handler, then this code kicks in and rather than returning parameters from the body broken out, it returns the body itself as a single arg.  This makes much more difficult to create decorators.

There's a similar condition (but not problematic for me) also on line 263 above.

The original change doesn't describe why.  It's just "connexion 2.0" in the blame.

I would propose removing the 'or has_kwargs' so that parameters are the same whether you break them out as named arguments in the handler or accept any named argument in the handler.  Thoughts?

Reply:
@ddurham2 I have tried to untagle the code in different parts of connexion without too much luck. I am afraid I can't help you. Due to various issues, like this one, we're forced to move away from `connexion` thus it is no longer my top priority to invest my time here.

Although, I am tracking what's going on, and my original opinion on state of affairs in connexion had not changed. Without strong API contract **enforced** & **required** by `connexion` it will be hard to make `connextion`, `Flask` and `aiohttp` implementation compliant.
And, since that contract is not in place, it is close to impossible or, at the very least, really hard to ensure proper implementation. 
One example of such situation is handling this multipart, I believe I've already mentioned that, where files from request are assumed to be `dist-ish` structure and as long as `dict-ish` structure is returned from `Flask` and `aiohttp` there little thing that makes a difference and that's implementation of that `dict-ish` being. TBH & AFAIR `connexion` requires that being to just be `iterable` or have `get` method which is rather poor expectation (all hail strong typed languages where weird stuff like that has lower chance to occur :/ ).

Reply:
If it were up to me, I'd have ditched that PR and focus on work to prepare strong internal API contract toward making `aiohttp` and `Flask` plugins.

Reply:
Did you find an alternative to connexion?

Reply:
There are alternatives but concept is a mirror reflection, essentially it is code that comes first. 
I am spending my time playing around with strongly typed self-creation :) now, it is available in my repositories ( [axion](https://github.com/kornicameister/axion/) ). It's no place to talk about it, but if you decide to check it out, let me know. It's mess from place to place but maybe sometime later I will clean it all up.